scenarios/happypath/build/build_application.go

```diff
@@ -107,7 +107,33 @@ func Application(ctx context.Context, cfg config.Config) error {
 		}
 	}
 
+	pipelineRuns := job.GetPipelineRuns(ctx, cfg, defaults.App2Name, jobName)
+	for _, run := range pipelineRuns {
+		var targetTask *models.PipelineRunTask
+		tasks := job.GetPipelineRunTasks(ctx, cfg, defaults.App2Name, jobName, *run.RealName)
+		for _, t := range tasks {
+			if *t.Name == "details" {
+				targetTask = t
+			}
+		}
+		if targetTask == nil {
+			return errors.New("Tekton test task is not found!")
+		}
+
+		// Test tekton log output contain parameters and secrets
+		tektonLogContent := job.GetLogForPipelineStep(ctx, cfg, defaults.App2Name, jobName, *run.RealName, *targetTask.RealName, "test-tekton")
+		if !strings.Contains(tektonLogContent, Secret1Value) {
+			return errors.New("Tekton test does not contain SecretValue")
+		}
+
+		if !strings.Contains(tektonLogContent, "github.com") {
+			return errors.New("Tekton test does no conaint github.com (should be printed from known_hosts)")
+		}
```

Comment:
Would it be more correct to check that any of lines contains the substring, e.g. with a method [Any](https://github.com/equinor/radix-common/blob/cabe99c6819cfd4b549e8ead9cad3eff355eea70/utils/slice/utils.go#L38) ?